README.md

```diff
@@ -61,6 +61,15 @@ make
 bin/fluent-bit -i cpu -o stdout -f 1
 ```
 
+To build on Windows
+
+```powershell
+cd build
+cmake -D FLB_HTTP_SERVER=No -D FLB_RELEASE=Yes -D FLB_DEBUG=No ..
```

Comment:
FWIW you can speed up the build speed by 2x by setting `-G NMake Makefiles`.
CMake is surprisingly slow at Visual Studio project generation.

For details, please refer to the commit message of ad0d67aa61d758b9f3f3061a36af4cbea1fa4fec.